MapboxCoreMaps.podspec

```
@@ -22,6 +22,6 @@ Pod::Spec.new do |m|
 
   m.vendored_frameworks = 'MapboxCoreMaps.xcframework'
 
-  m.dependency 'MapboxCommon', '11.0.0'
+  m.dependency 'MapboxCommon', '11.0.1'
```

We should start using the `~>` operator for common:

```suggestion
  m.dependency 'MapboxCommon', '~> 11.0'
```

---

See https://github.com/mapbox/mapbox-maps-internal/issues/879#issuecomment-820505836